rope/base/project.py

```diff
@@ -224,9 +224,22 @@ def __init__(
         super().__init__(fscommands)
         self.ignored = _ResourceMatcher()
         self.file_list = _FileListCacher(self)
+
+        extra_ignores = [
+            p.pathlib.relative_to(self.root.pathlib)
+            for p in self.get_python_path_folders()
+            if p.pathlib.is_relative_to(self.root.pathlib)
+            and "site-packages" in p.pathlib.parts
+        ]
+        for p in extra_ignores:
+            # I'm using this approach, because self.prefs.add after _init_prefs
+            # does not work for some reason.
+            prefs["ignored_resources"].append(str(p))
```

Comment:
That's my second attempt, and i need feedback here.

Is it good approach, or maybe some other ideas?